tests/test-cases/Group23-VIC-Machine-Service/23-06-VCH-Certificate.robot

```diff
@@ -71,7 +68,7 @@ Verify Certificate
 
 
 Verify Certificate Not Found
-    Output Should Contain        no certificate found for VCH
+    Output Should Contain        No certificate found for VCH
```

Comment:
Is this intentional change? because I remember a pr which changed all error messages to start with lower case character.

Reply:
Fixed, had a local inconsistency that was failing on the lowercased version.